README.md

```diff
@@ -57,6 +58,7 @@
 | [Auto-encoder Based Data Clustering](http://nlpr-web.ia.ac.cn/english/irds/People/lwang/M-MCG_EN/Publications/2013/CFS2013CIARP.pdf) |  ABDC  | CIARP 2013 |  [Pytrorch](https://github.com/KellerJordan/Autoencoder-Clustering) |
 | [Learning Deep Representations for Graph Clustering](https://www.aaai.org/ocs/index.php/AAAI/AAAI14/paper/viewFile/8527/8571) |    |  AAAI 2014 | [python](https://github.com/quinngroup/deep-representations-clustering) |
 |  [Deep Adaptive Image Clustering](http://openaccess.thecvf.com/content_ICCV_2017/papers/Chang_Deep_Adaptive_Image_ICCV_2017_paper.pdf) |    |  ICCV 2017||
+| [Deep Embedded Self-Organizing Map: Joint Representation Learning and Self-Organization](http://florentfo.rest/files/ESANN-2019-DeepEmbeddedSOM-full-paper.pdf) | DESOM | ESANN 2019 | [Keras](https://github.com/FlorentF9/DESOM) |
```

Comment:
This paper seems not very related to this repo :)

Reply:
DESOM is a (self-organized) deep clustering algorithm, but ok if this repo targets only "pure" clustering (in this family of algorithms there is also SOM-VAE and VarPSOM that are available on github)